include/sot/core/device.hh

```diff
@@ -93,6 +93,7 @@ namespace dynamicgraph {
       void setVelocitySize(const unsigned int& size);
       virtual void setVelocity(const dg::Vector & vel);
       virtual void setSecondOrderIntegration();
+      virtual void setNoIntegration();
```

Comment:
enum ControlInput is a member of the Device class. 
setSecondOrderIntegration and setNoIntegration are needed in the AbstractSotExternalInterface class, which is independent from the Device and defines the API for simulators and robots.

I don't think using an enum whose values are defined in a different class is a good idea for the Abstract API